arrow-core/src/main/kotlin/arrow/core/extensions/setk.kt

```diff
@@ -48,13 +56,19 @@ interface SetKEq<A> : Eq<SetK<A>> {
     else false
 }
 
-@extension
+@Deprecated(
+  "Typeclass instance have been moved to the companion object of the typeclass.",
+  ReplaceWith("Show.setK<A>()", "arrow.core.extensions", "arrow.core.extensions"),
```

Comment:
```suggestion
  ReplaceWith("Show.set<A>()", "arrow.core.set", "arrow.typeclasses.Show"),
```

You need to use fully qualified packages in the `ReplaceWith` statement. `SetK` is getting removed so we're not going to add an extension on `Show` for it.